frontend/src/router/index.js

```diff
@@ -72,6 +76,9 @@ const routes = [
         path: 'coverageStatusCheck',
         name: 'CoverageStatusCheck',
         component: CoverageStatusCheck,
+        beforeEnter: (to, from, next) => {
```

Comment:
This could be a `router.beforeEach()` with a switch statement on `to.name` to check the permissions. Would remove some of the duplicate `beforeEnter:` boilerplate we're going to end up with but it's a little more breakable if route names change. 
No real preference.

Reply:
I agree it would be less boilerplate but I didn't want to have to check hardcoded route names in a global beforeEach when each route already has the right context.